rest/data/commit_queue.go

```diff
@@ -131,8 +131,11 @@ func getPatchInfo(ctx context.Context, settings *evergreen.Settings, githubToken
 		return "", nil, nil, nil, errors.Wrap(err, "getting GitHub PR diff")
 	}
 
-	// fetch the latest config file
-	config, patchConfig, err := model.GetPatchedProject(ctx, settings, patchDoc, githubToken)
+	// Fetch the latest config file.
+	// Temporarily set a high timeout.
```

Comment:
Seems like the plan is to make this permanent if it succeeds right? If so can we mention [EVG-20924](https://jira.mongodb.org/browse/EVG-20924) in the comment here with a todo to make it untemporary if it works?

Reply:
true, i might just remove the word temporary since i'll just remove it if it doesn't work as part of the ticket